docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3122,7 +3136,36 @@ $$div[tag-content="banner"][min-length="400"]
 
 This rule will remove all the `div` elements, whose code contains the substring `banner` and the length of which exceeds `400` characters.
 
-**Exceptions**
+### Pseudo-classes
+
+#### `:contains()`
+
+##### Syntax
+```
+:contains(unquoted text)
+```
+or
+```
+:contains(/reg(ular )?ex(pression)?/)
+```
+
+:::note Compatibility
+
+`:-abp-contains()` and `:has-text()` are synonyms for `:contains()`.
+
+:::
+
+:::info Compatibility
+
+The `:contains()` pseudo-class is supported by AdGuard for Windows, Mac, and Android, **running CoreLibs version 1.13 or later**.
+
+:::
+
+Requires that the element's inner HTML contains the specified text or matches the specified regular expression.
+
+As with the `tag-content` and `wildcard` attributes, specifying a `:contains()` pseudo-class on a non-leaf selector is not supported.
```

Comment:
Could you please add a warning admonition here? This one's a pretty serious limitation.